.env.test

```diff
@@ -1,5 +1,6 @@
 NETWORK='testnet'
 BTC_CONFIRMATIONS=100
+RSK_MINIMUM_CONFIRMATION=10
```

Comment:
```suggestion
RSK_PEGOUT_MINIMUM_CONFIRMATIONS=10
```

Reply:
"I guess this isNewestStatus field should be used at the API level to ensure we have the latest, right?"
Yes. I had to do this because we will be searching for pegouts waiting for confirmation and waiting for signatures, and we need to have a way to mark them as 'process' in case we find a match in the bridge status, so in the next query we ignore them.